scalableminds/webknossos
# Access to Datasets on foreign Datastores

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- set up two instances with each standalone-datastore
- activate the feature in the config
- On Instance1: go to 'Datasets' -> 'edit' -> set Dataset public
   - again go to 'edit' -> 'general' -> copy text from the `Allow usage in other webknossos-instances using this text`-field
- On Instance2: go to 'Datasets' -> 'add Dataset' -> Tab 'add foreign Dataset'
   - paste the text copied previously into the field and then click on 'add Dataset'
   - test if you can view the Dataset
   - test if you can click on 'edit': the first tab should be disabled

### Issues:
- fixes #2945 

------
~~- [ ] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)~~
~~- [ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
